spec/unit/puppet_pal_2pec.rb

```diff
@@ -134,20 +114,6 @@
           end
           expect(result).to eq(10)
         end
-
-        it 'instantiates definitions in the given code string' do
```

Comment:
Same thing here - should change to being a test to call a function - change 'plan' to 'function'